Source/WebCore/html/SearchInputType.cpp

```diff
@@ -206,16 +206,16 @@ bool SearchInputType::sizeShouldIncludeDecoration(int, int& preferredSize) const
 {
     ASSERT(element());
     preferredSize = element()->size();
-    return true;
+    return element()->hasAttributeWithoutSynchronization(sizeAttr);
```

Comment:
I'm assuming this change is due to your offline comment about "the width of an <input type=search> now matches <input> (per spec, unless the size attribute is set)".

Can we add a link to the spec here?